src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Onramps style guide
+contentType: page
+template: basicDoc
+topics:
+  - onramps
+---
+
+import apmApmSummaryOnrampsGif from 'images/apm_screenshot-full_apm-summary-onramps-gif.gif'
+
+import apmOnrampsIssuesExample from 'images/apm_screenshot-crop_onramps-issues-example.webp'
+
+import apmEmptyStateExample from 'images/apm_screenshot-crop_empty-state-example.webp'
+
+import apmActiveStateExample from 'images/apm_screenshot-crop_active-state-example.webp'
+
+import apmServiceLevelsExample from 'images/apm_screenshot-crop_service-levels-example.webp'
+
+
+Our onramps content is the help text that appears in the UI when you click the **?** icon. The purpose of in-product help is to help users understand the UI, interpret performance data, and New Relic solutions with their system problems. Onramps provide more detail than a [tooltip](https://one-core.newrelic.com/components/tooltip/). Ideally, they give users essential info so they can stay in the product and keep working, rather than switching over to read the docs. 
+
+<img
+    title=""
+    alt=""
+    src={apmApmSummaryOnrampsGif}
+/>
+
+## Content guidelines
+
+* Be clear and concise. Contextual help should be clear and concise. Avoid using jargon or technical terms that users may not understand.
+* Use second-person point of view to address the reader. Address the user as "you", such as "We couldn't find your data." 
+* Answer the "why" and "how" questions. For example, a user might ask themselves, "Why is this important to me?" or "How do I use this information to improve my app performance?" Write content to help answer these questions, to understand the larger context of their application. For example, a user can learn about how APM and logs relate to each other, or how APM and infrastructure relate to each other.
+* Provide additional context to clarify ambiguous or confusing areas in the product. For example, if there is something that is not clearly defined in the UI, we can provide additional information to help users understand it.
+* Help users get more out of New Relic (and increase our revenue). When users are able to use New Relic more effectively, they are more likely to renew their subscriptions and recommend us to others. This, in turn, helps us to make more money. Include information that helps users:
+  * Understand what a feature is used for, and the benefits of the feature beyond the existing heading or other content. 
+  * Perform specific tasks.
+  * Understand what to do next. Are there recommended follow-on actions? The Apdex troubleshooting onramp provides a nice example for this. It tells users: "Compare the Apdex chart with other charts (like Web transaction time and Error rate) to look for the causes for the Apdex change."
+
+## FAQs
```

Comment:
I'd like us to not use the word FAQ in our docs, including in style guide.

In terms of restructuring, I think you have two types of content a bit mixed under this FAQ:

* Rules-type content:bold/italics
    * On-ramp titles
    * Length
    * Bold/italics
    * Links
    * Tile states
    * UI navigation info
* More philosophical info, which may actually belong above with the other philosphical info
   * When is onramps contgent needed?
   * Level of detail

A few ideas for how to restructure it:

* Just lose the FAQ header and make the questions all h2s
* Move the philosphical ones up into the earlier section, and group the others together here under a Rules-type heading
* Move all of them into a table to make the structure a bit more skimmable